rt/transports/http/src/main/java/org/apache/cxf/transport/http/HTTPConduit.java

```diff
@@ -714,6 +715,11 @@ public void close(Message msg) throws IOException {
                 }
             }
         } finally {
+            OutputStream os = msg.getContent(OutputStream.class);
```

Comment:
This is very strange (I will try to find out what is happening) but this change if needed belongs to `HttpClientHTTPConduit`

Reply:
Ok - i'll update the branch :)

Reply:
Branch is updated.

Running some tests to ensure the ephemeral port issue remains tamed.

Reply:
Appears good to me after moving the code.